source/installation/index.html

```diff
@@ -6,6 +6,25 @@
 toc: true
 ---
 
+<p>
+Home Assistant can be installed in 4 flavors:
+</p>
+<ul>
+    <li><em>Home Assistant Operating System</em>: Minimal Operating System optimized to power Home Assistant. Comes with Supervisor pre-installed. Recommended for new users.</li>
+    <li><em>Home Assistant Supervised</em>: Container based installation with our Supervisor to manage Home Assistant Core and Add-Ons.</li>
```

Comment:
```suggestion
    <li><em>Home Assistant Supervised</em>: Container based installation with our Supervisor to manage Home Assistant Core and add-ons.</li>
```

Reply:
I think we should avoid "Container-based installation". A container is an important driver, but we want to avoid confusion with the actual container-based installation type.

In general, this is considered a highly advanced installation type, so I think this should move lower down the list.

Reply:
It does requires a container engine to be useful, so I think mentioning the term container does make sense. Maybe we can put it to the end to weaken it?

I agree its highly advanced and we should not encourage people using it. But I kinda like to keep the order, as it is logically more to less...

How about adding some words to make it clear that this is an advanced installation? E.g. `Manual`: 

```suggestion
    <li><em>Home Assistant Supervised</em>: Manual installation of the Supervisor to manage Home Assistant Core and add-ons (using containers).</li>
```

Reply:
I really want to ditch containers, we have deliberately not put it into the ADRs as well.